tools/common-tools/src/demos/repl/components/fields/DerivationModeField.js

```diff
@@ -1,8 +1,8 @@
 // @flow
 import React from "react";
 import Select from "react-select";
-import { getAllDerivationModes } from "@ledgerhq/live-common/lib/derivation";
-import type { DerivationMode } from "@ledgerhq/live-common/lib/derivation";
+import { getAllDerivationModes } from "@ledgerhq/coin-framework/derivation";
+import type { DerivationMode } from "@ledgerhq/coin-framework/derivation";
```

Comment:
No need for /lib/ in these paths? (I actually don't know)

Reply:
Nope, it works without :)
If I correctly understood last works from Dex/Core team, the `lib` part from the import is unnecessary and undesired.